src/runtime/task/task_queue.cpp

```diff
@@ -114,4 +128,14 @@ void task_queue::enqueue_internal(task *task)
     tls_dsn.last_worker_queue_size = increase_count();
     enqueue(task);
 }
+
+const metric_entity_ptr &task_queue::queue_metric_entity() const
+{
+    CHECK_NOTNULL(_queue_metric_entity,
```

Comment:
How about outputing the name of the queue to distinguish which queue miss initialization?

Reply:
OK, I'll fix the checks for all related entities.